content/docs/azure/azureEndtoEnd.md

```diff
@@ -0,0 +1,171 @@
++++
+title = "End-to-End Pipeline Deployment on Azure"
+description = "An end-to-end guide to using the azurepipeline example"
+weight = 50
++++
+## Introductions
+### Overview of Azure and AKS
+
+Microsoft Azure is an open, flexible, enterprise-grade cloud computing platform running on Microsoft infrastructure. The platform has various services, including (SERVICES). 
+
+The [Azure CLI]((https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)) is a set of tools that you can use to interact with Azure from the command line.
+
+Azure Kubernetes Service (AKS) on Azure allows you to deploy containerized applications, within which you describe the resources your application needs, and AKS will manage the underlying resources automatically. This workflow is especially efficient at scale.
+### The overall workflow
+This guide will take you through using your Kubeflow deployment to build a Machine Learning pipeline on Azure. This guide uses a sample pipeline to detail the process of creating an ML workflow from scratch. You will create and run a pipeline that processes data, trains a model, and then registers and deploys that model as a webservice. 
```

Comment:
Change capital letters to lower case: machine learning. See the [style guide](https://www.kubeflow.org/docs/about/style-guide/) for hints on capitalization and other style guidelines.